reactnative-todo/src/components/common/specials.js

```diff
@@ -1,5 +1,12 @@
 import React, { useState, Fragment } from 'react';
```

Comment:
Please chance filename to `boolean` as that aligns with the callback it maps to.